src/pages/en/install/auto.md

```diff
@@ -33,18 +33,60 @@ yarn create astro
 
-## 2. Install Dependencies
+### Choose a Starter Template
+You will then see a short list of starter templates to choose from: 
+- `Just the basics`: A great starter template for anyone wanting to explore Astro.
+- `Blog`, `Documentation`, `Portfolio`: opinionated themes for specific use-cases.
+- `Completely empty`: A template that just includes the bare minimium to get started.
+
+Use the arrow keys (up and down) to navigate to the template you want to install, then press return (enter) to submit.
+
+### Install Dependencies (optional)
+The wizard will offer to run an `install` command for you at this time, which is optional.
+
+> If you do not do so at this time, you will need to [install dependencies](/en/install/auto#2-install-dependencies) after the wizard has finished, before starting your project.
+
+### Install any Official Astro Integrations (optional)
+You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Turbolinks, Partytown, Sitemap) by running `astro add --yes`
+
+To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle selected/unselected. You can select multiple items at once, or you can continue without selecting any items.
+
+
+When you are satisfied with your selection, press return (enter) to submit. 
+
+>🟢 Selected integrations will display a solid (green) circle. (All integrations are **unselected by default**.) 
+>
+>⚠️ If every item is unselected, then no integrations will be added at this time. 
+
+
+These integrations, and any [Astro community integrations](https://astro.build/integrations), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide).
 
-When the `create-astro` install wizard is complete, you should see some recommended instructions on your screen to follow that will help you complete setup and start your new project. 
+### See Your Astro Config File
 
-The only required step remaining is to install your project's dependencies using a package manager like npm:
+After selecting your integrations to add, you should see the following terminal message followed by **the entire contents of the `astro.config.mjs` file** that Astro will include with your project:
+
+```bash
+Astro will make the following changes to your config file:
+```
+
+At this time, you can verify that any integrations have been added to your project configuation, as well as any required `install` actions have run.
+
+
+### Initialize a .git Repository (optional)
+
+At the final step of the wizard, you will now be offered to initialize a git repository in your new directory, if you plan to use the tool [Git](https://git-scm.com/) in your project. This is optional.
```

Comment:
```suggestion
At the final step of the wizard, you will now be offered to initialize a git repository in your new directory. This is optional, but useful if you plan to use the tool [Git](https://git-scm.com/) for your project.
```

Reply:
Yes, much better, though I'm not much liking the stiltedness of my "At the final step of the wizard, you will now be offered to..." (ugh.) So, this makes it better, but I'll make the whole thing better in another pass, too.  :P

Reply:
reworded.